src/pages/AdminDashboard/CreateRoomConfirmation.tsx

```diff
@@ -21,16 +30,29 @@ export default function CreateRoomConfirmation() {
         return `${premiumVideoPagePrefixUrl}${routerUrl}`
     }
 
+    const classes = useStyles()
+
     const createDiscussion = () => {
-        dispatch(newRoom())
+        dispatch(resetRoomCreated())
     }
 
     return (
         <>
             <Typography variant="h5" component="h1">
-                {t('confirmation.heading', { roomName: 'toto' })}
+                {t('confirmation.heading', { roomName: createdRoom.name })}
             </Typography>
-            <Paper>{t('confirmation.message', { roomName: 'toto' })}</Paper>
+            <Box m={2} />
+            <Paper elevation={0} className={classes.paper}>
+                {t('confirmation.message', {
+                    hostName: '', //todo not return yet by api,
```

Comment:
I think I saw @HugoGresse fix on this one

Reply:
fixed in https://github.com/Instant-Visio/InstantVisio-WebApp/pull/518

Reply:
API deployed, getRooms now return the hostName

Reply:
hmm, I'm not sure it solve this through

Reply:
i just saw it now. `hostName` is not saved in firestore. Is it planned. We need it when we edit a room

Reply:
hostName are saved in firestore, on creation or edit

Reply:
my bad

Reply:
is only add if we set `sendAt` (and if we have `destinations` and `participants`).  `sendAt` is for reminder?

Reply:
yes, `sendsAt` is to create reminder, alongside hostName and destinations. It will be better to createReminderthis way too to prevent sending invitation to destination. 
No sendsAt with destinations = invite participant directly
sendsAt with destinations = create reminder. 

i'll update createRoom to save hostName no matter the case though.

Reply:
sendsAt is an array of seconds timestamp too, so you can create multiple reminders at once.

Reply:
There's an opened issue on that: #477

Reply:
API has been deployed and now always save the hostName